ftests/core/src/main/java/org/commonjava/indy/ftest/core/content/GroupMetadataMergeWhenSeqNewHostedAddedInMultiHostedGroupTest.java

```diff
@@ -0,0 +1,242 @@
+        try (final InputStream stream = client.content().get( hostedA.getKey(), path ))
+        {
+            assertContent( IOUtils.toString( stream ), repoAContent );
+        }
+
+        HostedRepository hostedB = createHostedAndStorePath( repoB, path, repoBContent );
+        try (final InputStream stream = client.content().get( hostedB.getKey(), path ))
+        {
+            assertContent( IOUtils.toString( stream ), repoBContent );
+        }
+
+        HostedRepository hostedC = createHostedAndStorePath( repoC, path, repoCContent );
+        try (final InputStream stream = client.content().get( hostedC.getKey(), path ))
+        {
+            assertContent( IOUtils.toString( stream ), repoCContent );
+        }
+
+        HostedRepository hostedD = createHostedAndStorePath( repoD, path, null );
+
+        Group g = new Group( "test", hostedA.getKey(), hostedB.getKey(), hostedD.getKey() );
+        g = client.stores().create( g, "adding group", Group.class );
+
+        logger.debug( "\n\nGroup constituents are:\n  {}\n\n", StringUtils.join( g.getConstituents(), "\n  " ) );
+
+        /* @formatter:off */
+        final String mergedContent1 =
+                "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
+                "  <metadata>\n" +
+                "    <groupId>org.foo</groupId>\n" +
+                "    <artifactId>bar</artifactId>\n" +
+                "    <versioning>\n" +
+                "      <latest>2.0</latest>\n" +
+                "      <release>2.0</release>\n" +
+                "      <versions>\n" +
+                "        <version>1.0</version>\n" +
+                "        <version>2.0</version>\n" +
+                "      </versions>\n"  +
+                "      <lastUpdated>20160722164334</lastUpdated>\n" +
+                "    </versioning>\n" +
+                "</metadata>\n";
+        /* @formatter:on */
+        try (final InputStream stream = client.content().get( group, g.getName(), path ))
+        {
+            assertContent( IOUtils.toString( stream ), mergedContent1 );
+        }
+
+        List<StoreKey> constituents = Arrays.asList(hostedA.getKey(), hostedB.getKey(),hostedC.getKey(), hostedD.getKey());
+        g.setConstituents( constituents );
+
+        client.stores().update( g, "test update" );
+        g = client.stores().load( StoreType.group, g.getName(), Group.class  );
+        logger.debug( "\n\nGroup constituents are:\n  {}\n\n", StringUtils.join( g.getConstituents(), "\n  " ) );
+
+        waitForEventPropagation();
+
+        /* @formatter:off */
+        final String mergedContent2 =
+                "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
+                "  <metadata>\n" +
+                "    <groupId>org.foo</groupId>\n" +
+                "    <artifactId>bar</artifactId>\n" +
+                "    <versioning>\n" +
+                "      <latest>3.0</latest>\n" +
```

Comment:
This test is currently failing in CI because the latest is the old version from hostedRepoB (?) (version 2.0). It doesn't go far enough through the assertions to find out if the `<version>3.0</version>` line is in the merged output...but it seems like this test might express the problem.

I'm really not sure why the first ftest didn't fail in CI though.